src/main/java/com/work/management/entity/Project.java

```diff
@@ -0,0 +1,50 @@
+package com.work.management.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class represents a single row in Projects Table.
+ */
+
+@Entity
+@Table(name = "PROJECTS")
+@Data
+@Builder
+@AllArgsConstructor
+@EqualsAndHashCode
+public final class Project {
```

Comment:
Same here...remove equals and Haschode...and add noArgs constructor...because if you are making allArgs constructor..then compiler won't provide noArgs constructor by default....you have to explicility make it...same for all DTo's..enityies and resources.